tests/cloudpickle_test.py

```diff
@@ -2321,6 +2323,22 @@ def __type__(self):
         o = MyClass()
         pickle_depickle(o, protocol=self.protocol)
 
+    @pytest.mark.skipif(
+        sys.version_info < (3, 6, 0),
+        reason="Dict determinism is a lost cause in Python < 3.6")
```

Comment:
With 3.5 EOL, it does not seem worth putting any effort into this. Downstream tools that care about determinism have likely been 3.6+ anyway.